### ISSUE TITLE
Incorrect Text on the "Topics for Web Accessibility Presentations and Training" Page

### ISSUE THREAD

**Issue by jfc3-dol** [w3c/wai-develop-training#34]
While reading through the "Topics for Web Accessibility Presentations and Training" page
https://www.w3.org/WAI/teach-advocate/accessibility-training/topics/#authoring ). I noticed it had incorrect text/grammar.

I highlighted the word "the", in the sentence below, which I believe should be "they", to make more sense in the sentence. The text can be found under the "Suggestions for Speakers" section, which is part of the "9. Authoring Tool Accessibility and ATAG" part of the page.

Activity/Demonstration: Explore how accessible authoring tools are, and how well **the** produce web content.

**Comment by remibetin** [comment]
@jfc3-dol Thanks for noticing this typo! The page has been corrected: https://www.w3.org/WAI/teach-advocate/accessibility-training/topics/#authoring
